src/main/java/mate/academy/springboot/swagger/util/SortParser.java

```diff
@@ -3,19 +3,25 @@
 import java.util.ArrayList;
 import java.util.List;
 import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
 
+@Component
 public class SortParser {
-    // products?count=20&page=1&sortBy=price:ASC;title:ASC
-    public static Sort parse(String sortBy) {
+    private static final int FIELD_INDEX = 1;
+    private static final int DIRECTION_INDEX = 0;
+    private static final String SEMICOLON = ";";
+    private static final String COLON = ":";
```

Comment:
FIELD_INDEX and DIRECTION_INDEX are examples of good naming
but SEMICOLON and COLON aren't good, it's like to write ONE instead of FIELD_INDEX,
names should be descriptive, and answer the question "for what do we use this constant?"